include/macros.h

```diff
@@ -24,9 +24,20 @@
 
 #define GET_ACTIVE_CAM(play) ((play)->cameraPtrs[(play)->activeCamId])
 
-#define SET_NEXT_GAMESTATE(curState, newInit, newStruct)    \
-    (curState)->nextGameStateInit = (GameStateFunc)newInit; \
-    (curState)->nextGameStateSize = sizeof(newStruct)
+#define STOP_GAMESTATE(curState)     \
+    do {                             \
+        GameState* state = curState; \
+                                     \
+        state->running = false;      \
+    } while(0)
+
+#define NEXT_GAMESTATE(curState, nextInit, nextSize) \
```

Comment:
Any reason not to use SET?

Reply:
Not really. I had kept the original macro so I could do each gamestate individually, and never update the name afterwards. I can add it back.